package.json

```diff
@@ -11,8 +11,8 @@
   "version": "0.9.0",
   "dependencies": {
     "Casper": "TryGhost/Casper#1.3.1",
-    "ghost": "0.9.0",
-    "ghost-s3-storage": "~0.2.2",
+    "ghost": "~0.10.1",
+    "ghost-s3-compat": "git://github.com/aorcsik/ghost-s3-compat.git",
```

Comment:
If we did end up going with this, can we get this pinned to a specific release/version of this dependencies so that nobody ever gets surprised by `master` of this repo not working for some reason?